src/library_int53.js

```diff
@@ -93,9 +93,8 @@ mergeInto(LibraryManager.library, {
     return HEAPU32[ptr>>2] + HEAPU32[ptr+4>>2] * 4294967296;
   },
 
-  // Converts the given signed 32-bit low-high pair to a JavaScript Number that can
-  // represent 53 bits of precision.
-  // TODO: Add $convertI32PairToI53Signaling() variant.
```

Comment:
Having a NaN-returning form of the function might not obviate the need for a signaling (exception throwing) variant of the function, so this TODO would still feel relevant even after a Checked version is added?